src/test/testLogger.ts

```diff
@@ -48,4 +51,9 @@ export class TestLogger implements Logger {
             })
         })
     }
+
+    public logLevelEnabled(logLevel: LogLevel): boolean {
+        const currentLevel = this.logLevel as LogLevel
+        return compareLogLevel(currentLevel, logLevel) >= 0
+    }
```

Comment:
Duplicate code from `WinstonToolkitLogger` because we don't have a base class :(